tests/QAN/pages/qanPage.js

```diff
@@ -14,6 +14,11 @@ module.exports = {
       queryAnalytics: '//div[a/span[text()="Query Analytics"]]',
     },
   },
+  elements: {
+    qanContainer: '//div[contains(@class, "query-analytics-data")]',
```

Comment:
please convert locators to css selectors. We have stated priority locat() > CSS > Xpath, see readme: https://github.com/percona/pmm-ui-tests/blob/main/CONTRIBUTING.md